eng/testing/linker/project.csproj.template

```diff
@@ -2,6 +2,13 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
+    <MonoAOTCompilerDir>{MonoAOTCompilerDir}</MonoAOTCompilerDir>
```

Comment:
Is it possible to only put these properties in here when they are needed? It feels like extra stuff that is unnecessary for the other legs.

Reply:
Because of the way we are writing out the projects to disk, the way to do this would be either to have a different template just for mono or to change the way we write the template. Both options seem a bit of overkill to me specially since the extra properties won't really affect the other legs build in any way if they are defined, so I would opt for simplicity and keeping as is unless you feel strongly about it.